lib/vmdb/util.rb

```diff
@@ -1,7 +1,10 @@
 module VMDB
   module Util
-    def self.http_proxy_uri
-      proxy = VMDB::Config.new("vmdb").config[:http_proxy] || {}
+    def self.http_proxy_uri(proxy_config = :default)
+      # TODO: (julian) This looks really messy. Need to make it look nicer.
+      vmdb_proxy = VMDB::Config.new("vmdb").config[:http_proxy]
+      return nil unless vmdb_proxy
+      proxy = vmdb_proxy[proxy_config] || {}
```

Comment:
Actually I would make this backward compatible unless, @Fryguy do we have tools in place to migrate settings?

To make it backward compatible, this should work:
```ruby
proxy = vmdb_proxy[proxy_config] || vmbd_proxy
```